modules/victoria-metrics/templates/values.yaml

```diff
@@ -1143,7 +1147,7 @@ crds:
 
 ## install prometheus operator crds
 prometheus-operator-crds:
-  enabled: false
+  enabled: true
```

Comment:
By installing these CRDs we can install prometheus specific resources and victoria metrics will create the relevant victoria metric resource. This is useful when installing helm charts that have defined scrapes and other prometheus configs.